packages/components/fields/select-field/README.md

```diff
@@ -57,6 +57,7 @@ export default Example;
 | `renderError`              | `Function`<br/>[See signature.](#signature-renderError)                                            |          |           | This function can return a message which will be wrapped in an ErrorMessage. It can also return null to show no error.&#xA;<br/>                                                                                                                                                                                                                                                         |
 | `isRequired`               | `boolean`                                                                                          |          |           | Indicates if the value is required. Shows an the "required asterisk" if so.                                                                                                                                                                                                                                                                                                              |
 | `touched`                  | `union`<br/>Possible values:<br/>`boolean[] , boolean`                                             |          |           | Indicates whether the field was touched. Errors will only be shown when the field was touched.                                                                                                                                                                                                                                                                                           |
+| `appearance`               | `union`<br/>Possible values:<br/>`'default' , 'quiet'`                                             |          |           | Indicates the appearance of the input.&#xA;Quite appearence is meant to be used with the `horizontalConstraint="auto"`.                                                                                                                                                                                                                                                                  |
```

Comment:
```suggestion
| `appearance`               | `union`<br/>Possible values:<br/>`'default' , 'quiet'`                                             |          |           | Indicates the appearance of the input.&#xA; Quiet appearence is meant to be used with the `horizontalConstraint="auto"`.                                                                                                                                                                                                                                                                  |
```

Reply:
@Rhotimee ah thank you for noticing! Fixed here - [ac7dc7a](https://github.com/commercetools/ui-kit/pull/2644/commits/ac7dc7a89da0d9565eba40086de74a23a5286085) ✅